queries.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is that file for testing purpose? If yes would it be better to put it in a tests folder?

Reply:
Added by mistake. Rmeoved